### PR TITLE
docs(readme): Inserting Angular Logo in Readme.md

### LINKED ISSUE #10513: docs: Angular Icon is missing in README.md
**What is the current behavior?**

The angular icon is missing from the readme file. 
Screenshot (31)_LI

"Architects.io" logo is also missing from the file but I think that it is an invalid URL.

**What is the expected behavior?**

Screenshot (30)_LI

**Additional Details**
I want to work on this issue

### PR DESCRIPTION
**Pull request by himanshu010**
Fixes #10513  

**What kind of change does this PR introduce?**
Improving Documentation

**Did you add tests for your changes?**
N/A

**Does this PR introduce a breaking change?**
N/A

**What needs to be documented once your changes are merged?**
N/A

### CODE REVIEW

**Comment by himanshu010** [comment]
> Any screenshot will be highly apperitiated.

Screenshot (32)

**Comment by snitin315** [comment]
@himanshu010 please sign CLA.

**Comment by himanshu010** [comment]
> @himanshu010 please sign CLA.

Done

File: README.md [hunk 1]
```diff
@@ -439,7 +439,9 @@ Before we started using OpenCollective, donations were made anonymously. Now tha
 
 <div align="center">
 
-[Google Angular Team](https://angular.io/), [Architects.io](https://architects.io/),
+<a href="https://angular.io/" target="_blank" title="JS framework"><img
+src="https://cdn.worldvectorlogo.com/logos/angular-icon-1.svg" height="30" alt="Angular"></a>
+, [Architects.io](https://architects.io/),
```
Comment (snitin315) [inline README.md]: This link is broken as well

Comment (himanshu010) [reply README.md]: Yeah, I know, but I am not able to find the correct link.

**Comment by sokra** [comment]
Thanks

**Pull request merged**